### DISCUSSION TITLE
Add MysqlDsn type

### DISCUSSION THREAD

**Discussion by aradkdj** [pydantic/pydantic#4620] [Feature Request]
It would be nice if you could add a MySQL DSN. I'm not entirely sure how to add it myself, but I could open a PR for it if you can give me some guidance

**Comment by samuelcolvin** [comment]
Realistically at this point, let's wait for V2. Should be pretty simple look at how other dsns are done.

**Reply by AmitArie** [reply]
@samuelcolvin do you know what is the expected date of V2? thx!

**Reply by samuelcolvin** [reply]
#4887

**Comment by aradkdj** [comment]
This seems resolved since #4990. Thank you guys!
